Source/WebKit/UIProcess/Automation/WebAutomationSession.cpp

```diff
@@ -2569,6 +2576,17 @@ Ref<WebAutomationSession::Debuggable> WebAutomationSession::protectedDebuggable(
     return m_debuggable;
 }
 
+void WebAutomationSession::logEntryAdded(const String& level, const String& text, const WallTime& timestamp, const String& type, const std::optional<String>& method)
+{
+    // FIXME Support getting source information
+    // https://bugs.webkit.org/show_bug.cgi?id=282978
+    String sourceString;
+
+    // FIXME Get browsing context handle and source info
+    // https://bugs.webkit.org/show_bug.cgi?id=282981
+    m_domainNotifier->logEntryAdded(level, sourceString, text, timestamp.secondsSinceEpoch().milliseconds(), type, method.value_or(emptyString()));
```

Comment:
so i think it'd be here that does the conversion from `MessageSource`/`MessageType`/`MessageLevel` to `String`

we could also have enum types for each inside the `Automation` domain, but perhaps that's not necessary for now